eslzArm/eslz-portal.json

```diff
@@ -1463,6 +1463,32 @@
                                 ]
                             }
                         },
+                        {
+                            "name": "esAzFwSku",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Select Azure Firewall Sku",
```

Comment:
Can we change the label from 'Sku' to 'tier' so we are using the same taxonomy in the regular firewall deployment experience?
![image](https://user-images.githubusercontent.com/16622613/132526858-88db4a4a-975c-45da-aa09-6bf9f4a667b6.png)

Reply:
DONE